src/components/v5/shared/SearchSelect/SearchSelect.tsx

```diff
@@ -153,7 +155,19 @@ const SearchSelect = React.forwardRef<HTMLDivElement, SearchSelectProps>(
                 )
               ) : (
                 <>
-                  {showEmptyContent && (
+                  {isDefaultItemVisible ? (
+                    <button
+                      type="button"
+                      className="text-sm hover:text-blue-400 flex items-center gap-2 min-h-[3.125rem]
+                      "
+                      onClick={() => onSelect?.(searchValue)}
+                    >
+                      <Avatar />
+                      <span className="max-w-[15.625rem] truncate">
+                        {searchValue}
+                      </span>
+                    </button>
+                  ) : (
                     <EmptyContent
```

Comment:
Didn't you just break `showEmptyContent` prop?

Reply:
in that case empty content shoudn't be display here. In other places empty content works fine.

Reply:
But in @CzarekDryl PR if the user pastes in the token address then the dropdown content is hidden